dpnp/dpnp_iface_counting.py

```diff
@@ -44,25 +44,38 @@
 __all__ = ["count_nonzero"]
 
 
-def count_nonzero(a, axis=None, *, keepdims=False):
+def count_nonzero(a, axis=None, *, keepdims=False, out=None):
```

Comment:
Could you add tests covering `out` parameter?

Reply:
and `keepdims` as well

Reply:
Done